src/data/content/design-system/componenti/header.yaml

```diff
@@ -82,7 +84,32 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              La lista dei casi d’uso possibili per ciascuna variante del componente header.
+
+              ### Slim Header
+
+              Lo _slim header_ è una versione più **compatta** e **flessibile** dell’header completa.
```

Comment:
Forse va riformulata la frase. 
@zetareticoli @danielaiozzo Lo slim header è più un pezzo possibile dell'header completa che un'alternativa però. È nato e di solito è sempre stato usato per indicare l'ente di appartenenza del progetto o della pagina, oppure l'ente superiore in gerarchia. Vedi il ns uso su Designers per es. Così come le Regioni nel tema Comuni. O il MUR nel tema scuole.

Reply:
Ok per me.